packages/mermaid-zenuml/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@mermaid-js/mermaid-zenuml",
+  "version": "0.0.1",
+  "description": "Zenuml integration for MermaidJS.",
+  "module": "dist/mermaid-zenuml.core.mjs",
+  "types": "dist/detector.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/mermaid-zenuml.core.mjs",
+      "types": "./dist/detector.d.ts"
+    },
+    "./*": "./*"
+  },
+  "keywords": [
+    "diagram",
+    "markdown",
+    "zenuml",
+    "mermaid"
+  ],
+  "scripts": {
+    "prepublishOnly": "pnpm -w run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mermaid-js/mermaid"
+  },
+  "author": "Peng Xiao",
+  "license": "MIT",
+  "standard": {
+    "ignore": [
+      "dist/**/*.js"
+    ],
+    "globals": [
+      "page"
+    ]
+  },
+  "dependencies": {
+    "@zenuml/core": "^3.0.0"
+  },
+  "devDependencies": {
+    "mermaid": "workspace:*"
```

Comment:
Fixed in PR https://github.com/mermaid-js/mermaid/pull/4405, except with:

`"peerDependencies": { "mermaid": "workspace:>=10.0.0"`, see discussion in https://github.com/mermaid-js/mermaid/pull/4405#discussion_r1195041114.

It's a bit weird, but it still works! (I've tested it with `pnpm pack`).

I'd still prefer it to be a `^10.x` instead of a `>=`, in case Mermaid v11 breaks something in the plugin system, but we can always fix that in a future release.